Makefile

```diff
@@ -185,3 +185,8 @@ push-manifests: prepare-workdir
 	cd $(WORKDIR)
 	@$(CONTAINER_ENGINE) run --rm -v $(WORKDIR):/workdir -w /workdir \
 		$(LINUX_IMAGE) /bin/bash -c "${SCRIPTS_DIR}/push_manifests.sh '$(GIT_USER_NAME)' $(GIT_USER_EMAIL) $(GIT_TOKEN) $(PR_OR_COMMIT_URL) $(DEPLOYMENT_REPO) $(DEPLOYMENT_BRANCH) $(WORKFLOW_ID) $(APPLICATION_ID) $(IMAGE_NAME) $(IMAGE_TAG)"
+
+remove-whitespaces:
+	@echo "Removing all whitespaces from application.properties files"
```

Comment:
just to be on the safe-side, let's add replace `whitespaces` with `trailing whitespaces`